src/SleepRepository.js

```diff
@@ -1,6 +1,10 @@
-if (typeof module !== 'undefined') {
-  var Sleep = require('../src/Sleep');
-}
+// Commented out if block in order for code to be compatible in browser. 
+// This currently breaks testing. 
+// Will refactor.
+
+// if (typeof module !== 'undefined') {
+//   let Sleep = require('../src/Sleep');
+// }
```

Comment:
This code needs to be commented out in order to work in browser, but by doing so our test breaks. We will need to refactor to be compatible both with Chai and the browser.